Support/Testing/Travis/before-script.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+set -eu
+
+if [[ "$TARGET" = "Android-ARM" ]]; then
+  android_tools="$(pwd)/android-sdk-linux/tools"
+  android_script="$android_tools/android"
+
+  if [ ! -f $android_script ] ; then
+    android_sdk_name="android-sdk_r24.4.1-linux"
+    wget "https://dl.google.com/android/$android_sdk_name.tgz"
+    tar -zxf "$android_sdk_name.tgz"
+
+    echo "y" | $android_script update sdk -u -a --filter android-21
+    echo "y" | $android_script update sdk -u -a --filter sys-img-armeabi-v7a-android-21
+
+    echo "no" | $android_script create avd --force -n test -t android-21 --abi armeabi-v7a
+  fi
+
+  $android_tools/emulator64-arm -avd test -no-skin -no-audio -no-window &
```

Comment:
This script should just have the part that installs the emulator. Starting it should be done separately.